modules/picard/fastqtosam/main.nf

```diff
@@ -0,0 +1,45 @@
+process PICARD_FASTQTOSAM {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::picard=2.27.4" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/picard:2.27.4--hdfd78af_0' :
+        'quay.io/biocontainers/picard:2.27.4--hdfd78af_0' }"
```

Comment:
Have you tried using picard-slim as you suggested on Slack?

Reply:
I didn't. I was thinking most pipelines will already have the picard image downloaded for one tool or another, and this would cause an extra image to be loaded